ludo/src/main/java/de/uniks/ludo/LudoUtil.java

```diff
@@ -5,6 +5,16 @@
 
 public class LudoUtil {
 
+    private static final String CONTROLLER_TEST = "controller.test.enabled";
+
+    public static boolean inControllerTest() {
+        return Boolean.parseBoolean(System.getProperty(CONTROLLER_TEST));
+    }
+
+    public static void enableControllerTest() {
+        System.setProperty(CONTROLLER_TEST, Boolean.toString(true));
+    }
```

Comment:
Why use system property if a static variable can achieve the same?

Reply:
With the current setup this might be true, but using a system property allows for enabling the option using parameters instead of a method call.